tests/integration/events.bats

```diff
@@ -111,3 +111,56 @@ function teardown() {
   run eval "grep -q 'test_busybox' events.log"
   [ "$status" -eq 0 ]
 }
+
+@test "events oom " {
+  # XXX: currently cgroups require root containers.
+  requires root
+  init_cgroup_paths
+
+  # Set some initial known values
+  if [ "$CGROUP_UNIFIED" == "yes" ]; then
+  DATA=$(cat <<EOF
+  "memory": {
+    "limit": 33554433,
+    "kernel": 33554433,
```

Comment:
No need to specify kernel limit here, it serves no purpose (and is ignored for v2 anyway)

Reply:
Yes, I just copy from v1, I'll remove `kernel` and `swappiness`.